kong/db/schema/entities/consumers.lua

```diff
@@ -13,15 +13,15 @@ return {
     {
       username = {
         description =
-        "The unique username of the Consumer. You must send either this field or custom_id with the request.",
+        "The unique username of the Consumer. You must send at least one of username or custom_id with the request.",
         type = "string",
         unique = true
       },
     },
     {
       custom_id =
       {
-        description = "Stores the existing unique ID of the consumer.",
+        description = "Stores the existing unique ID of the consumer. You must send at least one of username or custom_id with the request",
```

Comment:
Missing final period:

```suggestion
        description = "Stores the existing unique ID of the consumer. You must send at least one of username or custom_id with the request.",
```